src/data/agile-quiz.ts

```diff
@@ -175,6 +175,17 @@ const agileQuiz = [
       "Planning poker is an estimation and planning technique in the Agile development model. It is used to estimate the development effort required for a user story or a feature.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/",
   },
+  {
+    Question:
+      "In which environment does Agile works great?",
+    Answer: "It works great in dynamic environments where there is a potential for changing or evolving requirements.",
+    Distractor1: "It works great in static environments.",
+    Distractor2: "It works great in customer environment.",
+    Distractor3: "It works great in a team environment.",
```

Comment:
I think team environment is a little vague here. Let's change this answer 👍 
```suggestion
    Distractor3: "Agile has been proven to not work in any good environment",
```